pkg/predicate/predicate.go

```diff
@@ -51,3 +58,37 @@ func (GenerationChangedPredicate) Update(e event.UpdateEvent) bool {
 	}
 	return true
 }
+
+// Skips events that have labels matching a selector defined in watches.yaml
+func EventFilter(eventLabels map[string]string, r ResourceFilterPredicate) bool {
```

Comment:
This should be `func (r ResourceFilterPredicate) Event filter(eventLabels map[string]string) bool`

Reply:
Also should probably be lowercase `eventFilter`, this is an implementation detail and not something we should expose outside of this package